Makefile

```diff
@@ -0,0 +1,7 @@
+test: lint
+	@python3 -m unittest
+
+lint:
+	@flake8
```

Comment:
Looking at our .travis.yml we'd want to have lint be both:
    - autopep8 -r --aggressive --diff --exit-code .
    - flake8

(IIRC, autopep8 gives you a nicer diff suggestion when your syntax is incorrect, rather than just telling you it is wrong).

And then we can change the .travis.yml script to be:

script:
  make test

I don't know if we'd want to be cautious and add "sudo apt install flake8 make" just to be extra cautious that Make is available on the build machines.